docs/application-connector/10-04-connector-service-troubleshooting.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Errors while generating or renewing a certificate
+type: Troubleshooting
+---
+
+## Invalid Certificate Signing Request (CSR)
+
+If you try to generate a client certificate, you may get this error:
+```json
+{
+  "code":403,
+  "error":"CSR: Invalid common name provided."
+}
+```
+
+This error is caused by applying wrong subject info to your Certificate Signing Request.  
+To ensure it was generated properly, check the values returned by the Connector Service with the call that fetched CSR info:
```

Comment:
The call doesn't fetch CSR details only.  Read this to understand the context. 

https://kyma-project.io/docs/components/application-connector/#tutorials-get-the-client-certificate-get-the-csr-information-and-configuration-details-from-kyma